.devcontainer/almalinux/devcontainer.json

```diff
@@ -1,3 +1,5 @@
+// SPDX-FileCopyrightText: 2025 Contributors to the Media eXchange Layer project https://github.com/dmf-mxl/mxl/contributors.md
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
C and C++ style comments are not an official part of JSON and rejected by many JSON parsers/validators. I don't like the sidecar .license files because they bloat the directory view, but I think we probably should for formats that don't support comments...